src/test/unit/components/GameCard.test.tsx

```diff
@@ -2,34 +2,54 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import { GameCard } from '../../../components/GameCard';
 
-// const img = "path-to-image"
-
-describe('Rendering GameCard', () => {
-  beforeEach(() => {
-    render(
-    <GameCard 
-        gameTitle='Dragon Age' 
-        gameDescription='Surprisingly few dragons.'
-        gameImage=''
-     />);
-  });
+describe('Rendering Game GameCard', () => {
+  
 
   it('renders GameCard', () => {
+    render(
+      <GameCard 
+          gameTitle='Dragon Age'
+          gameDescription='Surprisingly few dragons.'
+          gameImage={["//images.igdb.com/igdb/image/upload/t_thumb/jrumdkrrox09wbtblrka.jpg"]} 
+          gamePlatforms={["PS3"]} 
+          gameGenres={["RPG"]} 
+          gameUniverses={["Dragon Age"]} 
+          gameCompanies={["BioWare"]} 
+          gameReleaseDate={["Nov 03, 2009"]}
+          />);
     const text = screen.getByText(/Add to Library/i);
     expect(text).toBeInTheDocument();
   })
-  // TODO: Once functionality in GameCard is working, uncomment these tests
-//   it('renders game title', () => {
-    // const title = screen.getByText(/Dragon Age/i);
-    // expect(title).toBeInTheDocument();
-//   })
 
-//   it('renders game description', () => {
-    // const description = screen.getByText(/Surprisingly few dragons./i);
-    // expect(title).toBeInTheDocument();
-//   })
+  it('renders game title', () => {
+    render(
+      <GameCard 
```

Comment:
Can we render this just once in a beforeEach loop? Doesn't look like we're changing anything across renders